contribs/pom.xml

```diff
@@ -90,6 +90,7 @@
 		<module>parking</module>
 		<module>protobuf</module>
 		<module>pseudosimulation</module>
+		<module>railsim</module>
```

Comment:
Please add this new contrib to the list of contribs in https://github.com/matsim-org/matsim-libs/blob/master/.github/workflows/verify-push.yaml